libs/api-client-bear/src/gooddata-node.ts

```diff
@@ -1,17 +1,22 @@
-// (C) 2007-2020 GoodData Corporation
+// (C) 2007-2022 GoodData Corporation
 
 import fetchCookie from "fetch-cookie";
 import nodeFetch from "node-fetch";
 
 import { factory, SDK } from "./gooddata";
 
+// AbstractFactory always reinitializes fetch-cookie. This way, when factory initialization
+// happens between requests, new instance of fetch-cookie and uderlying cookie jar
+// will be used.
+const abstractFactoryNode = (config?: any) => factory(fetchCookie(nodeFetch as any))(config);
```

Comment:
Should we add this to the browser entry point as well to keep the API symmetrical (even though it is not needed there)?

Reply:
Good point, I will add it there as well.